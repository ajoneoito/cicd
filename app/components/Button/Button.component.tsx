import {StyleSheet, Text, TouchableOpacity} from 'react-native';
import React from 'react';

const Button = () => {
  return (
    <TouchableOpacity style={style.btn}>
      <Text style={style.btnText}>Button</Text>
    </TouchableOpacity>
  );
};

export default Button;

const style = StyleSheet.create({
  btn: {
    backgroundColor: '#28AF51',
    padding: 12,
    borderRadius: 4,
    marginTop: 8,
  },
  btnText: {
    textAlign: 'center',
    color: '#fff',
    fontSize: 14,
    fontWeight: '500',
  },
});
