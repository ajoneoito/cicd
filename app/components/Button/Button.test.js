import React from 'react';
import renderer from 'react-test-renderer';
import Button from './Button.component';

test('renders correctly', () => {
  const tree = renderer.create(<Button />).toJSON();
  expect(tree).toMatchSnapshot();
});