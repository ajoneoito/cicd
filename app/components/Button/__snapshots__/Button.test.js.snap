// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessibilityState={
    {
      "busy": undefined,
      "checked": undefined,
      "disabled": undefined,
      "expanded": undefined,
      "selected": undefined,
    }
  }
  accessibilityValue={
    {
      "max": undefined,
      "min": undefined,
      "now": undefined,
      "text": undefined,
    }
  }
  accessible={true}
  collapsable={false}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    {
      "backgroundColor": "#28AF51",
      "borderRadius": 4,
      "marginTop": 8,
      "opacity": 1,
      "padding": 12,
    }
  }
>
  <Text
    style={
      {
        "color": "#fff",
        "fontSize": 14,
        "fontWeight": "500",
        "textAlign": "center",
      }
    }
  >
    Button
  </Text>
</View>
`;
